src/actions/index.js

```diff
@@ -0,0 +1,9 @@
+// action constants 
+export const UPDATE_PRODUCTS = 'UPDATE_PRODUCTS' ;
+
+// action creators
+export const uploadProducts = (products) => {
```

Comment:
Have a look at `redux-actions` NPM module.

It gives us a function called createAction. Lets use that here